jekyll/_cci2/docker-layer-caching.md

```diff
@@ -1,58 +1,85 @@
 ---
 layout: classic-docs
-title: "Enabling Docker Layer Caching"
-short-title: "Enabling Docker Layer Caching"
+title: "Docker layer caching overview"
 description: "How to reuse unchanged cache layers in images you build to reduce overall run time"
-categories: [optimization]
-order: 70
 contentTags:
   platform:
   - Cloud
   - Server v4.x
   - Server v3.x
 ---
 
-Docker layer caching (DLC) can reduce Docker image build times on CircleCI. DLC is available on
-the [Free and above](https://circleci.com/pricing/) usage plans (credits are charged per run job) and on installations of [CircleCI server](https://circleci.com/enterprise/).
+Use Docker layer caching (DLC) to reduce Docker image build times on CircleCI. DLC is available on all CircleCI plans, and credits are charged per job run.
```

Comment:
do you wanna link to where they can see the credits per job run?

Reply:
I found this FAQ that we could link to, but this content might also need some updates: the 200 credits for DLC, and the second paragraph. https://circleci.com/docs/credits/#charge-for-docker-layer-caching Any thoughts @sebastian-lerner ?

Reply:
sorry i missed this, i was thinking https://circleci.com/pricing/